tests/udf/test_blobfinder.py

```diff
@@ -30,10 +31,12 @@ def test_refinenemt():
     assert (rx < x) and (rx > (x - 1))
 
 
+@pytest.mark.xfail(reason="FIXME update blobfinder UDF to new UDF interface")
```

Comment:
This needs to be removed; it is the cause for the decrease in coverage. I think you can just revert the whole file to the state it is in `master` and things should be fine.

Reply:
Done! :-)